.rubocop.yml

```diff
@@ -0,0 +1,20 @@
+AllCops:
+  NewCops: enable
+  Exclude:
+    - db/**/*
+    - tmp/**/*
+    - Gemfile
+    - Gemfile.lock
```

Comment:
Doesn't this imply that all the old files would be  tested if we ran rubocop?

Reply:
Sorry, why? This `Exclude` is ignoring those files.